### DISCUSSION TITLE
ja/blog/4-scroll-code-wordpress-hestia

### DISCUSSION THREAD

**Discussion by giscus** [atman-33/computing-atman-v4-mk2#1] [General]
# ja/blog/4-scroll-code-wordpress-hestia

This is an article for those using the WordPress theme "Hestia. "When embedding source code on a post page, is there a concern that it might wrap and become hard to read?

http://localhost:4200/ja/blog/4-scroll-code-wordpress-hestia

**Comment by atman-33** [comment]
test comment...
